composer.json

```diff
@@ -10,12 +10,12 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^8.0",
         "friendsofphp/php-cs-fixer": "^3.0",
         "nette/utils": "^3.2",
-        "slevomat/coding-standard": "^6.4.1 || ^7.0",
-        "squizlabs/php_codesniffer": "^3.6",
-        "symplify/easy-coding-standard": "^10.0 <10.2.4"
+        "slevomat/coding-standard": "^8.0",
+        "squizlabs/php_codesniffer": "^3.9",
+        "symplify/easy-coding-standard": "^12.1"
```

Comment:
With lowest version allowed by `^12.1`  there is an error:

```
 [ERROR] Call to undefined method Symplify\EasyCodingStandard\Config\ECSConfig::configure()                             
```

This function was probably added in some bugfix version :-D :raised_eyebrow:  . So the constrain must be to at least this version.

Reply:
Changed to `^12.1.4`